tests/dragonfly/replication_test.py

```diff
@@ -458,3 +463,19 @@ async def check_expire(key):
         await check_expire("k")
         await check("GETEX k EX 100", r"PEXPIREAT k (.*?)")
         await check_expire("k")
+
+        # Check SDIFFSTORE turns into DEL and SADD
+        await c_master.sadd("set1", "v1", "v2", "v3")
+        await c_master.sadd("set2", "v1", "v2")
+        await skip_cmd()
+        await skip_cmd()
+        await check("SDIFFSTORE k set1 set2", [r"DEL k", r"SADD k v3"])
+
+        # Check SINTERSTORE turns into DEL and SADD
+        await check("SINTERSTORE k set1 set2", [r"DEL k", r"SADD k (.*?)"])
+
+        # Check SMOVE turns into SREM and SADD
+        await check("SMOVE set1 set2 v3", [r"SREM set1 v3", r"SADD set2 v3"])
+
+        # Check SUNIONSTORE turns into DEL and SADD
+        await check("SUNIONSTORE k set1 set2", [r"DEL k", r"SADD k (.*?)"])
```

Comment:
You already checked this

Reply:
There is no other place SINTERSTORE is checked
above checked SINTERSTORE